concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -66,8 +66,8 @@ pub struct HashBytes([u8; SHA256 as usize]);
 impl HashBytes {
     /// Construct HashBytes from a slice.
     /// This will succeed if and only if the length of the slice is 32.
-    pub fn new(bytes: &[u8]) -> Fallible<Self> {
-        let buf: Result<[u8; 32], _> = bytes.try_into();
+    pub fn new(bytes: &[u8]) -> anyhow::Result<Self> {
+        let buf: anyhow::Result<[u8; 32], _> = bytes.try_into();
         let buf = buf.context("HashBytes::new passed slice of incorrect length.")?;
```

Comment:
I don't think this context is necessary. That will already be the message from the `try_into`.

Wouldn't just `let buf = bytes.try_into()?` or `let buf = bytes.try_into().context("HashBytes::new. ...)?` work without ugly type annotations.